pages/bookmarks.jsx

```diff
@@ -37,7 +37,7 @@ export default React.createClass({
     return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
   },
   getAnonymousContent() {
-    return (<p>Are you Mozilla Staff? <button className="btn btn-link inline-link" onClick={this.handleSignInBtnClick}>Sign in</button>.</p>);
+    return (<p>Are you Mozilla Staff? <a href="javascript:void(0)" onClick={this.handleSignInBtnClick}>Sign in</a>.</p>);
```

Comment:
semantics becomes muddled here: it's not a *navigation* link, users are not intentionally going to "a url", they are intentionally logging in, which happens to use redirects but they couldn't care less really, so in this case a button is fine, as long as it can be tabbed to for accessibility purposes, so I think what we want here is to preserve the button, but make sure it has a `tabIndex=0` so that it's a keyboard-navigable element in the browser.